src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.shared.cs

```diff
@@ -0,0 +1,28 @@
+namespace CommunityToolkit.Maui.Core.Views;
+
+public static partial class DrawingViewService
+{
+	/// <summary>
+	/// Get image stream from points
+	/// </summary>
+	/// <param name="points">Drawing points</param>
+	/// <param name="desiredSize">The desired dimensions of the generated image. The image will be resized proportionally.</param>
+	/// <param name="lineWidth">Line Width</param>
+	/// <param name="strokeColor">Line color</param>
+	/// <param name="background">Image background</param>
+	/// <param name="token"><see cref="CancellationToken"/></param>
+	/// <returns>Image stream</returns>
+	public static ValueTask<Stream> GetImageStream(IList<PointF> points, Size desiredSize, float lineWidth, Color strokeColor, Paint? background, CancellationToken token = default) =>
```

Comment:
What do you think about moving parameters in Options class?
So new method signature looks like this:
```suggestion
	public static ValueTask<Stream> GetImageStream(IList<PointF> points, ImageOptions options, CancellationToken token = default)
```

In this case we may easily extend image options without breaking changes.

Reply:
Great idea!

Reply:
I need to fix some unit tests and move some files around but what do you think to the latest set of changes?